en/08_Changelogs/5.4.0.md

```diff
@@ -164,6 +164,8 @@ The changes include:
 - [`SSTemplateParser`](api:SilverStripe\View\SSTemplateParser) has been deprecated. It will be renamed to `SilverStripe\TemplateEngine\SSTemplateParser`.
 - [`TemplateIteratorProvider`](api:SilverStripe\View\TemplateIteratorProvider) has been deprecated. It will be renamed to `SilverStripe\TemplateEngine\TemplateIteratorProvider`.
 - [`TemplateParser`](api:SilverStripe\View\TemplateParser) has been deprecated. It will be renamed to `SilverStripe\TemplateEngine\TemplateParser`.
+- [`ElementalAreaController::removeNamespacesFromFields()`](api:DNADesign\Elemental\Controllers\ElementalAreaController::removeNamespacesFromFields()) has been deprecated. It will be removed without equivalent functionality to replace it.
```

Comment:
Is being moved rather than removed?

Reply:
It's a completely different method now - it doesn't take in an array of data and it doesn't output an array. I think it's more honest to consider it a new different method.
Otherwise we'd have to say "has been moved and the method signature has changed and you can't give it data anymore" or something.